ai-ml/jupyterhub/helm/jupyterhub/jupyterhub-values-dummy.yaml

```diff
@@ -25,6 +25,17 @@ proxy:
 singleuser:
   startTimeout: 1200 # 20 mins to spin up a notebook server for GPU including the image pull
   profileList:
+    - display_name: Elyra (CPU)
+      description: "Elyra Notebooks | Karpenter Autoscaling"
+      kubespawner_override:
+        image: 975050295866.dkr.ecr.us-east-1.amazonaws.com/elyra:3.15.0 # Replace with image pushed to ECR
```

Comment:
I've added you to the Data on EKS AWS account. You can build and deploy the image with the instructions below and update the image in the code with public.ecr.aws/data-on-eks/elyra-jupyter:latest.

```bash
aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws/data-on-eks

docker build -t elyra-jupyter .

docker tag elyra-jupyter:latest public.ecr.aws/data-on-eks/elyra-jupyter:latest

docker push public.ecr.aws/data-on-eks/elyra-jupyter:latest
```